src/lib.rs

```diff
@@ -1227,6 +1227,10 @@ pub(crate) const HASH_SEC_PARAM: usize = 128;
 mod error;
 pub use error::{Error, FrostResult};
 
+/// A module defining traits for implementing convenient encoding and decoding to/from bytes.
+pub mod serialization;
+pub use serialization::{FromBytes, ToBytes};
```

Comment:
```suggestion
pub use serialization::{self, FromBytes, ToBytes};
```

Should work I think?

Reply:
It does, but my actually intended behavior was to have the module private, and only expose the two traits. Not sure why I made it public in the first place!